packages/react-sdk/src/stitches/to-css.ts

```diff
@@ -2,9 +2,10 @@ import type { CSS } from "./stitches";
 import type { StyleProperty, StyleValue, CssRule, Breakpoint } from "../css";
 import { DEFAULT_FONT_FALLBACK, SYSTEM_FONTS } from "@webstudio-is/fonts";
 
-export const toValue = (value: StyleValue): string => {
+export const toValue = (value: StyleValue | undefined): string => {
+  if (value === undefined) return "";
   if (value.type === "unit") {
-    return value.value + (value.unit === "number" ? "" : value.unit);
+    return value.value + value.unit;
```

Comment:
are you saying that we now have a unit type that has an empty string value.unit instead of number?